src/main/kotlin/ee/carlrobert/codegpt/actions/toolwindow/DeleteConversationAction.kt

```diff
@@ -0,0 +1,36 @@
+package ee.carlrobert.codegpt.actions.toolwindow
+
+import com.intellij.icons.AllIcons
+import com.intellij.openapi.actionSystem.ActionUpdateThread
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.ui.Messages
+import ee.carlrobert.codegpt.actions.ActionType
+import ee.carlrobert.codegpt.actions.editor.EditorActionsUtil.registerAction
+import ee.carlrobert.codegpt.conversations.ConversationsState
+import ee.carlrobert.codegpt.telemetry.TelemetryAction
+import ee.carlrobert.codegpt.ui.OverlayUtil
+
+class DeleteConversationAction(private val onDelete: Runnable) :
+  AnAction("Delete Conversation", "Delete single conversation", AllIcons.Actions.GC) {
+  init {
+    registerAction(this)
+  }
+
+  override fun update(event: AnActionEvent) {
+    event.presentation.isEnabled = ConversationsState.getCurrentConversation() != null
+  }
+
+  override fun actionPerformed(event: AnActionEvent) {
+    if (OverlayUtil.showDeleteConversationDialog() == Messages.YES && event.project != null) {
```

Comment:
Why show a dialog if event.project != null? 😅
```suggestion
    if (event.project != null && OverlayUtil.showDeleteConversationDialog() == Messages.YES) {
```